### PR TITLE
Autosave returns, with undo/redo support back to page load time

### PR DESCRIPTION
**Pull request by boutell**

### CODE REVIEW

**Comment by boutell** [comment]
The "publish changes" button will be back! It just has no function right now, until draft/live arrives.

File: modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue [hunk 1]
```diff
@@ -121,6 +114,7 @@
             :icon-only="true"
             @click="redo"
           />
+          <span class="apos-admin-bar__status">{{ status() }}</span>
```
Comment (abea) [inline modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue:117]: Unless I'm missing something, this should be a computed property.

**Review by abea — CHANGES_REQUESTED** [review]

**Review by abea — APPROVED** [review]

**Comment by abea** [comment]
@boutell The other thing I forgot to mention was that it seems like we're ending up with a lot of editing state logic being stored in here. I'm not sure there's a great alternative. A single `status` property that alternates `editing/retrying/saved` would probably be over-engineering.

I think my best thought regarding this is that after the Vue 3 update we separate the code and use Composition API to keep things cleaner.

**Comment by boutell** [comment]
The admin bar is pretty much responsible for the editing state, but looking forward to Vue 3 ways of factoring things that are unnecessarily complicated.

**Pull request merged**